suite/tests/CMakeLists.txt

```diff
@@ -6533,6 +6533,10 @@ if (NOT ANDROID AND AARCHXX)
       code_api|sample.memtrace_simple
       code_api|sample.memval_simple
       PROPERTIES LABELS RUNS_ON_QEMU)
+    if (DEBUG)
```

Comment:
It only passes in DEBUG?  That is odd; add a XXX comment pointing to an issue # covering fixing for release.

Reply:
I would include a comment here as well as a reader is not going to have seen the earlier one.  Your suggestion seems fine or even shorter `XXX i#1806: client.flush currently fails in release build.`